evm/gas-service.js

```diff
@@ -0,0 +1,414 @@
+'use strict';
+
+const { ethers } = require('hardhat');
+const {
+    getDefaultProvider,
+    utils: { formatEther, parseEther },
+    Contract,
+    BigNumber,
+} = ethers;
+const { Command, Option } = require('commander');
+const {
+    printInfo,
+    printWalletInfo,
+    printWarn,
+    mainProcessor,
+    prompt,
+    getGasOptions,
+    wasEventEmitted,
+    isValidAddress,
+    validateParameters,
+} = require('./utils');
+const { addBaseOptions } = require('./cli-utils');
+const IAxelarGasService = require('@axelar-network/axelar-gmp-sdk-solidity/interfaces/IAxelarGasService.json');
+const { getWallet } = require('./sign-utils');
+
+async function getGasInfo(env, chain, destinationChains) {
+    const apiUrl = env === 'testnet' ? 'https://testnet.api.axelarscan.io/gmp/getFees' : 'https://api.axelarscan.io/gmp/getFees';
+
+    return Promise.all(
+        destinationChains.map((destinationChain) => {
+            let feeType = 0;
+            let blobBaseFee = 0;
+
+            switch (destinationChain.toLowerCase()) {
+                case 'ethereum-sepolia':
+
+                // eslint-disable-next-line no-fallthrough
+                case 'ethereum': {
+                    feeType = 0;
+                    blobBaseFee = env === 'testnet' ? 50187563959 : 1;
```

Comment:
query this dynamically from an RPC (we can do it in a separate task)

Reply:
I moved this to the config, so we can have an upper boundary there.